### PR TITLE
Prepare release 1.1.0.0

### PR DESCRIPTION
**Pull request by chshersh**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,12 +3,27 @@
 `relude` uses [PVP Versioning][1].
 The changelog is available [on GitHub][2].
 
-## Unreleased
+## 1.1.0.0 — Apr ??, 2022
```
Comment (chshersh) [inline CHANGELOG.md:6]: TODO: change the date when the time comes

**Comment by thomasjm** [comment]
Hello, is there an ETA on this by any chance? I'd love to see a GHC 9.2.2 compatible release on Hackage. Thanks!

**Comment by donatello** [comment]
Please release GHC 9.2 compatible version to hackage! Thank you in advance!

**Review by vrom911 — APPROVED** [review]

**Comment by vrom911** [comment]
[`relude-1.1.0.0`](https://hackage.haskell.org/package/relude) has been released to hackage. Thanks a lot for your patience 🙂

**Pull request merged**